phpcs.xml.dist

```diff
@@ -1,26 +1,65 @@
 <?xml version="1.0"?>
-<ruleset name="Coding Standards for doctrine">
-    <description>Coding Standards for doctrine.</description>
-
+<ruleset>
     <arg name="basepath" value="."/>
     <arg name="extensions" value="php"/>
     <arg name="parallel" value="80"/>
     <arg name="cache" value=".phpcs-cache"/>
     <arg name="colors" />
 
-    <!-- Ignore warnings and show progress of the run -->
-    <arg value="np"/>
+    <!-- Ignore warnings, show progress of the run and show sniff names -->
+    <arg value="nps"/>
 
     <file>lib</file>
     <file>tests</file>
 
-    <rule ref="Doctrine"/>
```

Comment:
This is wrong, it must stay otherwise no CS. 😆

Reply:
Fixed.